### PR TITLE
fix: remove duplicate value i18n

### LINKED ISSUE #8546: [CAL-1600] Duplicate values in en.json
### Issue Summary

There are many duplicate values in the locales/en/common.json file. for example key booking_submitted 

<sub>[CAL-1600](https://linear.app/calcom/issue/CAL-1600/duplicate-values-in-enjson)</sub>

### PR DESCRIPTION
**Pull request by Udit-takkar**
## What does this PR do?

Fixes https://github.com/calcom/cal.com/issues/8546

I just used this simple script to find all duplicate values

```

const valueSet = new Set();

for (const key in common) {
    if (valueSet.has(common[key])) {
      console.log(`Duplicate value found in: ${key}`);
    } else {
      valueSet.add(common[key]);
    }
}
```

Edit: I have added duplicate check in next.config.js as suggested by hariom

Screenshot 2023-06-07 at 5 22 41 PM

**Environment**: Staging(main branch) / Production

## Type of change

- Chore (refactoring code, technical debt, workflow improvements)

### CODE REVIEW

**Comment by Udit-takkar** [comment]
NEXT: remove all these keys from other languages JSON file

File: apps/web/pages/settings/security/two-factor-auth.tsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ const TwoFactorAuthView = () => {
   const isCalProvider = user?.identityProvider === "CAL";
   return (
     <>
-      <Meta title={t("2fa")} description={t("2fa_description")} />
+      <Meta title={t("2fa")} description={t("password_description")} />
```
Comment (hariombalhara) [inline apps/web/pages/settings/security/two-factor-auth.tsx:39]: We actually need to fix the text for `2fa_description` here IMHO. This is 2 factor page and not password page.

File: apps/web/public/static/locales/en/common.json [hunk 2]
```diff
@@ -485,15 +478,11 @@
...
   "in_person_meeting": "In-person meeting",
-  "attendee_in_person": "In Person (Attendee Address)",
   "in_person": "In Person (Organizer Address)",
   "link_meeting": "Link meeting",
-  "phone_call": "Attendee Phone Number",
-  "your_number": "Your phone number",
   "phone_number": "Phone Number",
   "attendee_phone_number": "Attendee Phone Number",
   "organizer_phone_number": "Organizer Phone Number",
-  "host_phone_number": "Your Phone Number",
```
Comment (hariombalhara) [inline apps/web/public/static/locales/en/common.json:496]: Is it possible that the text is stored in DB and we won't find it in code search so we consider it unused?

Though if that is being done, we should fix that. But just wanted to put this question out there.

File: apps/web/pages/settings/security/two-factor-auth.tsx (hunk 1 above)
Comment (Udit-takkar) [reply apps/web/pages/settings/security/two-factor-auth.tsx:39]: what text should i use here?

**Review by hariombalhara — CHANGES_REQUESTED** [review]
It's an awesome PR !!
Left a comment or two.
Apart from that, I think we should add the check in next.config.js itself which can throw a warning(may be an error) when it detects duplicate values in en/common.json.

It would avoid doing this work again.

File: apps/web/pages/settings/security/two-factor-auth.tsx (hunk 1 above)
Comment (PeerRich) [reply apps/web/pages/settings/security/two-factor-auth.tsx:39]: "Set up your Two-factor authentication"

**Comment by PeerRich** [comment]
@Udit-takkar can you reveisit this one? or is it too many conflicts?

**Comment by Udit-takkar** [comment]
@PeerRich fixing this PR

File: apps/web/next.config.js [hunk 3]
```diff
@@ -57,6 +57,20 @@ if (process.env.GOOGLE_API_CREDENTIALS && !validJson(process.env.GOOGLE_API_CRED
   );
 }
 
+const checkDuplicateValue = () => {
```
Comment (hariombalhara) [inline apps/web/next.config.js:60]:
```suggestion
const informAboutDuplicateTranslations = () => {
```

File: apps/web/next.config.js [hunk 4]
```diff
@@ -57,6 +57,20 @@ if (process.env.GOOGLE_API_CREDENTIALS && !validJson(process.env.GOOGLE_API_CRED
...
+  for (const key in englishTranslation) {
+    if (valueSet.has(englishTranslation[key])) {
+      console.warn("\x1b[33mDuplicate value found in:", "\x1b[0m", key);
+    } else {
+      valueSet.add(englishTranslation[key]);
+    }
+  }
+};
+
+checkDuplicateValue();
```
Comment (hariombalhara) [inline apps/web/next.config.js:72]:
```suggestion
informAboutDuplicateTranslations();
```

**Review by hariombalhara — APPROVED** [review]

**Pull request merged**